### DISCUSSION TITLE
Shipping with no tax class

### DISCUSSION THREAD

**Discussion by piloujp** [zencart/zencart#6359] [General]
Shipping cost with no tax class set is always considered as a non-taxable item in Zen Cart, but should it be?
I might be wrong, but for accounting reason it should be the opposite, always be considered as taxable. Final tax amount on invoice should include shipping tax.
If somebody could ask a professional accountant, it would be nice to have some clarifications.

**Comment by torvista** [comment]
Commenting as a EU store shipping in and outside the EU, the customer final invoice has a cost for products and shipping. All of that has tax levied or not.
So my shipping prices are with/without tax based on destination, same as the customer zones.
The exception, which we still handle manually/outside ZC, is for trade customers with an EU-vat code that can buy in the EU without tax. This will be handled at some point when I get round to wholesale pricing.

**Reply by piloujp** [reply]
Interesting, I know a little bit European system and it is relatively simple as you just have to apply tax (or not) to the invoice total.
In Japan, it is more complicated, everybody has to pay the VTA but if you register to the tax refund system then at the end of the year you can fill a sale tax refund form and get it back ... in theory. Since last year, they added a second registration so invoices you make are 'certified refundable' ... Now you can only ask for a VTA refund for certified invoices! And of course these registrations are not straight forward, you must submit to certain conditions.
In a way it is simpler for small business like mine, just have to include tax everywhere!
I wonder how it is in the state, I know a bit of different local taxes you add to make your final sale tax, but I wonder how it works when you sell from states with tax based on billing to states with tax based on shipping? Do you have to pay tax twice?

**Reply by drbyte** [reply]
In North America, "tax on shipping" is determined by destination. So the shipping rate is quoted without tax, and if tax is applicable based on zone (and that zone has a tax rate configured for it) then tax gets added to that shipping rate during checkout. No double-taxing.
Same with tax on products.

There are some exceptions/variations based on specific industries, but those rules are quite clearly defined in those industries and States where they apply.

**Comment by torvista** [comment]
I think this highlights why it's better to use a locale-based ERP to cover all these country-specific anomalies and just use ZC as a sales channel: the invoice from the ERP can be legally correct and just has to match the total money paid in ZC. Trying to drag ZC to match the ERP seems too much.

**Comment by piloujp** [comment]
@drbyte Thanks for this clarification, now I understand better ZC design.

@torvista I can't follow your thinking, ZC does not have to rival with ERP, it should just be able to have invoice with right total amount and tax so afterward accounting can be done easier. Plus, for small business, like one man personal business, ERP is too costly.
